Jieun/models/userDao.js

```diff
@@ -45,7 +51,26 @@ const isUserExisted = async (userId) => {
   }
 };
 
+const loginUser = async (email) => {
```

Comment:
```suggestion
const getUserByEmail = async (email) => {
```
dao는 Data Access Object의 약자로서 해당 부분에 작성되는 함수이름으로 loginUser는 적합하지 않습니다. CRUD(create, get, update, delete)와 자원의 조합으로 getUserByEmail과 같은 식으로 작성해주세요!